chunky/src/java/se/llbit/chunky/renderer/scene/PathTracer.java

```diff
@@ -389,6 +402,36 @@ public static boolean pathTrace(Scene scene, Ray ray, WorkerState state, int add
     return hit;
   }
 
+  private static Vector4 sampleEmitter(Scene scene, Ray ray, Grid.EmitterPosition pos, Random random) {
```

Comment:
Some javadoc on top of this method that explains what it does (i.e. a summary of this PR) and maybe a link to this PR as `@see` would be nice.